packages/connector-local/src/connector.ts

```diff
@@ -51,7 +52,7 @@ export default class LocalConnector implements IConnector {
     private engine: Engine<HTMLEvents>;
     private _href: string = '';
     private filesPattern: string[];
-    private watcher: chokidar.FSWatcher | null = null;
+    public watcher: chokidar.FSWatcher | null = null;
```

Comment:
Yes, because we are removing the serial, we can't not use `process.emit('SIGINT')`, so I'm using in tests `connector.watcher.emit('close');` and for that, the watcher has to be public

Reply:
You are right, fixed.